pkgs/misc/vscode-extensions/python/default.nix

```diff
@@ -75,6 +75,21 @@ in vscode-utils.buildVscodeMarketplaceExtension rec {
     # Patch `packages.json` so that nix's *ctags* is used as default value for `python.workspaceSymbols.ctagsPath`.
     substituteInPlace "./package.json" \
       --replace "\"default\": \"ctags\"" "\"default\": \"${ctagsDefaultsTo}\""
+
+
+    # Similar cleanup to what's done in the `debugpy` python package.
+    # This prevent our autopatchelf logic to bark on unsupported binaries (`attach_x86.so`
+    # was problematic) but also should make our derivation less heavy.
+    (
+      set -x
```

Comment:
```suggestion
```

Reply:
Done. Included in `95e30ee27ee8c18c659758d6b43b6816ada8a60f`.